### ISSUE TITLE
Kbox create pantheon fails after prompting for password (fresh install of 0.10.4)

### ISSUE THREAD

**Issue by pi-ron** [kalabox/kalabox#615]
``````
info: Finishing install
info: -- OK! 100% complete! --

info: Huzzah! Victory!
Tims-iMac:Sites timdaff$ kbox create pantheon
? Choose a Pantheon account. tim@nextentity.com.au
Your Pantheon session has expired. We need to reauth!
? Pantheon dashboard password (tim@nextentity.com.au) ********
? Pantheon dashboard password (tim@nextentity.com.au) ********
Unhandled rejection TypeError: Cannot read property 'account' of undefined
    at /usr/local/bin/node_modules/kalabox-app-pantheon/vendor/kalabox-app-pantheon/app/plugins/kalabox-plugin-pantheon/lib/client.js.jx:291:21
From previous event:
    at Client.reAuthSession (/usr/local/bin/node_modules/kalabox-app-pantheon/vendor/kalabox-app-pantheon/app/plugins/kalabox-plugin-pantheon/lib/client.js.jx:280:4)
    at Client.__request (/usr/local/bin/node_modules/kalabox-app-pantheon/vendor/kalabox-app-pantheon/app/plugins/kalabox-plugin-pantheon/lib/client.js.jx:348:19)
    at Client.getSites (/usr/local/bin/node_modules/kalabox-app-pantheon/vendor/kalabox-app-pantheon/app/plugins/kalabox-plugin-pantheon/lib/client.js.jx:673:15)
    at /usr/local/bin/node_modules/kalabox-app-pantheon/lib/create.js.jx:237:29
    at process._tickCallback (node.js:812:13)
From previous event:
    at Object.kbox.create.add.option.inquire.choices (/usr/local/bin/node_modules/kalabox-app-pantheon/lib/create.js.jx:236:12)
    at Object.utils.runAsync (/usr/local/bin/node_modules/inquirer/lib/utils/utils.js.jx:28:22)
    at Object.<anonymous> (/usr/local/bin/node_modules/inquirer/lib/utils/utils.js.jx:74:11)
    at Object.utils.runAsync (/usr/local/bin/node_modules/inquirer/lib/utils/utils.js.jx:28:22)
    at /usr/local/bin/node_modules/inquirer/lib/utils/utils.js.jx:50:13
    at tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at setDisposable (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5232:36)
    at AnonymousObservable.s [as _subscribe] (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5249:11)
    at AnonymousObservable.Rx.Observable.observableProto.subscribe.observableProto.forEach (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1975:19)
    at /usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:2448:21
    at tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at setDisposable (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5232:36)
    at AnonymousObservable.s [as _subscribe] (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5249:11)
    at AnonymousObservable.Rx.Observable.observableProto.subscribe.observableProto.forEach (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1975:19)
    at MergeObserver.handleSubscribe (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:3368:28)
    at MergeObserver.onNext (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:3374:16)
    at MergeObserver.tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at AutoDetachObserverPrototype.next (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5273:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1709:35)
    at InnerObserver.onNext (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:4809:14)
    at InnerObserver.tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at AutoDetachObserverPrototype.next (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5273:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1709:35)
    at InnerObserver.onNext (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:3408:91)
    at InnerObserver.tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at AutoDetachObserverPrototype.next (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5273:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1709:35)
    at scheduleItem (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:2960:16)
    at Scheduler.scheduleNow [as _schedule] (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1223:50)
    at Scheduler.Rx.Scheduler.schedulerProto.scheduleWithState (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:969:19)
    at JustSink.run (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:2965:36)
``````

**Comment by pirog** [comment]
@pi-ron im guessing your kalabox.json for that app was never created successfully. can you paste in its contents here?

**Comment by pi-ron** [comment]
This is on my home computer I'll dig it up asap.

**Comment by pirog** [comment]
nice! that would be very helpful @pi-ron. thanks for your awesome help as always.

**Comment by pi-ron** [comment]
Thanks for those new pull options!

**Comment by katypool** [comment]
@pirog I'm getting this error as well on kalabox 0.10.5 (mac)

Not sure about your question about the kalabox.json -- this error is happening on kbox create pantheon, after I get prompted for a password and before I tell it what site to pull down. No app has been created yet.

**Comment by pirog** [comment]
thanks for the report @katypool! 

going into `~/.kalabox/terminus/session`, deleting the offending session and then running `kbox create pantheon` again seems to resolve the issue.

i can replicate this now so moving into the current sprint.

**Comment by pirog** [comment]
@pi-ron @katypool i think this should be fixed in 0.10.6
